<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>414</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>414</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login to PMS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pms_resources.qrc">
    <normaloff>:/images/account_login.png</normaloff>:/images/account_login.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameBackground">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>112</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>152</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_username">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="plainText">
       <string>root</string>
      </property>
      <property name="placeholderText">
       <string>User name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_localhost">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="plainText">
       <string>localhost</string>
      </property>
      <property name="placeholderText">
       <string>IP/Host name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>160</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_pass">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>150</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="plainText">
       <string>1234</string>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_port">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>210</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="plainText">
       <string>3306</string>
      </property>
      <property name="placeholderText">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>220</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_connect">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>280</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:8px;
border: 1px solid rgb(210, 210, 210);
background-color: rgb(245, 245, 245);
}

QPushButton:hover {
color: rgb(255, 255, 255);
border-radius:8px;
border: 1px solid rgb(23, 120, 173);
background-color: #299ED6;
}

QPushButton:pressed {
border-radius:10px;
border: 1px solid #777777;
background-color: #222222;
}

QPushButton:checked {
border-radius:10px;
border: 1px solid rgb(150, 150, 150);
background-color: rgb(38, 38, 38);
}

</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>275</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Server status:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="serverStatus">
      <property name="geometry">
       <rect>
        <x>542</x>
        <y>275</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Not connected</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:0;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Welcome!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: 0;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Please log in to PMS using a user account assigned to you by the Administrator.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtID">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:5px;
border: 1px solid rgb(170, 170, 170);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(254, 254, 254), stop: 1 rgb(252, 252, 252));
}

QLineEdit:hover {
border-radius:5px;
border: 2px solid #299ed6;
background-color: rgb(255, 255, 255);
}

QLineEdit:pressed {
border-radius:5px;
border: 1px solid #777777;
}
</string>
      </property>
      <property name="placeholderText">
       <string>Your user account ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:5px;
border: 1px solid rgb(170, 170, 170);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(254, 254, 254), stop: 1 rgb(252, 252, 252));
}

QLineEdit:hover {
border-radius:5px;
border: 2px solid #299ed6;
background-color: rgb(255, 255, 255);
}

QLineEdit:pressed {
border-radius:5px;
border: 1px solid #777777;
}
</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Your password</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>190</y>
        <width>168</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCommandLinkButton {
color: rgb(50, 50, 50);
background-color:0;
border:0;
}

QCommandLinkButton:hover {
border-radius: 5px;
border-style: inset;
background-color: #FFFFFF;
border: 2px solid #299ed6;
}

QCommandLinkButton:pressed {
border-radius:5px;
border: 1px solid #fdfdfd;
background-color: #cccccc;
}

QCommandLinkButton:checked {
color: rgb(255, 255, 255);
border-radius:5px;
background-color: rgb(220, 220, 220);
}

</string>
      </property>
      <property name="text">
       <string>Database configuration</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>-1</y>
        <width>1</width>
        <height>350</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>432</width>
        <height>66</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>432</width>
        <height>66</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>432</width>
        <height>66</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/login_header.png);
background-repeat:0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <zorder>frame</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>plainTextEdit_username</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>plainTextEdit_localhost</zorder>
     <zorder>label_5</zorder>
     <zorder>plainTextEdit_pass</zorder>
     <zorder>plainTextEdit_port</zorder>
     <zorder>label_6</zorder>
     <zorder>pushButton_connect</zorder>
     <zorder>label_8</zorder>
     <zorder>serverStatus</zorder>
     <zorder>label_7</zorder>
     <zorder>label_9</zorder>
     <zorder>txtID</zorder>
     <zorder>txtPassword</zorder>
     <zorder>commandLinkButton</zorder>
     <zorder>frame_2</zorder>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(160, 160, 160);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 5px;
border: 1px solid rgb(150, 150, 150);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}

QPushButton:hover {
border-style: inset;
background-color: #FFFFFF;
border: 2px solid #299ed6;
}

QPushButton:pressed {
border: 1px solid rgb(140, 140, 140);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #999, stop: 1 #ddd);
}

#frame_5 {
border: none;
background-color: rgb(205, 205, 205);
}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: 0;</string>
        </property>
        <property name="text">
         <string>Remember login details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="styleSheet">
         <string notr="true">background-color: 0;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="icon">
         <iconset resource="pms_resources.qrc">
          <normaloff>:/images/check.png</normaloff>:/images/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="pms_resources.qrc">
          <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pms_resources.qrc"/>
 </resources>
 <connections/>
</ui>
